src/main/java/io/kabanero/api/restricted/GitHubEndpoints.java

```diff
@@ -51,4 +59,43 @@ public User getUserInfo() throws IOException {
         return new UserService(client).getUser();
     }
 
+    @POST
+    @Path("/team/{wantedTeamId}/member/{github_username}")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response addTeamMember(@PathParam("wantedTeamId") int wantedTeamId, @PathParam("github_username") String githubUsername) throws IOException, ApiException, GeneralSecurityException {
+        try{
+            UserProfile userProfile = UserProfileManager.getUserProfile();
+            String token = userProfile.getAccessToken();
+            GitHubClient client = new GitHubClient();
+            client.setOAuth2Token(token);
+            TeamService teamService = new TeamService(client);      
+            teamService.addMember(wantedTeamId, githubUsername);
+            //teamService.addMember is a void method so we don't know if the user was removed from the team so return 202 instead of 200
```

Comment:
update comment, you mention the `addMember` function and then later metntion "if the user was removed from the team" which isn't very clear.